{
use_imm: true,
#Define number of models
number_of_models: 1,
#Setting Markov transition probabilites
markov_trans_prob_0_0: 1,
#Configuration of models used
IMM0_motion_model: BrownianMotion,
    IMM0_cosxx: 0.4,
    IMM0_cosyy: 0.4,
    IMM0_cosw: 0,
    IMM0_cosvxx: 0,
    IMM0_cosvyy: 0,
    IMM0_use_process_noise: true,
    IMM0_process_noise_level: 1,
    IMM0_qsvxx: 0,
    IMM0_qsvyy: 0,
    IMM0_qsxx: 0.0081,
    IMM0_qsyy: 0.0081,
send_imm_debug_information: true,
}
